File: C++/C/115.cpp
/*
とある世界では、今日は 12 月 D 日です。
D=25 なら Christmas, D=24 なら Christmas Eve, D=23 なら Christmas Eve Eve, D=22 なら Christmas Eve Eve Eve と出力するプログラムを作ってください。
*/
#include <iostream>
using namespace std;
int main()
{
    int D;
    cin >> D;
    if (D == 25)
        cout << "Christmas\n";
    else if (D == 24)
        cout << "Christmas Eve\n";
    else if (D == 23)
        cout << "Christmas Eve Eve\n";
    else
        cout << "Christmas Eve Eve Eve\n";
}

File: ruby/A/115.rb
# とある世界では、今日は 12 月 D 日です。
# D=25 なら Christmas, D=24 なら Christmas Eve, D=23 なら Christmas Eve Eve, D=22 なら Christmas Eve Eve Eve と出力するプログラムを作ってください。
d = gets.to_i
if d == 25
  puts "Christmas"
elsif d == 24
  puts "Christmas Eve"
elsif d == 23
  puts "Christmas Eve Eve"
else d == 22
  puts "Christmas Eve Eve Eve" end
# 02'30
